client/web/compose/src/components/PageBlocks/Configurator.vue

```diff
@@ -128,6 +128,16 @@
           >
             {{ $t('general.hidden.label') }}
           </b-form-checkbox>
+          <b-form-checkbox
+            v-if="block.kind !== 'Tabs'"
+            v-model="block.meta.namespaceID"
+            :value="namespace.namespaceID"
+            :disabled="alreadyGlobalBlock"
```

Comment:
Not sure we need the alreadyGlobalBlock.
You just need to properly provide the :value and @update events to toggle the property